plugins/poll/app/controllers/polls_controller.rb

```diff
@@ -61,18 +61,25 @@ def grouped_poll_results
     poll_name = params.require(:poll_name)
     user_field_name = params.require(:user_field_name)
 
-    begin
+    if Poll.find_by(post_id: post_id, name: poll_name).type == "irv"
```

Comment:
What if a poll in that post with that name doesn't exist?

Reply:
It should also maybe use the `.irv?` method you added on a poll?

Reply:
good catch(es)!.  resolved here: https://github.com/discourse/discourse/pull/27155/commits/b2c51d6f4a33a1432b94793c685b57513bd8c1cc